src/main/java/lotto/Client.java

```diff
@@ -0,0 +1,38 @@
+package lotto;
+
+import lotto.controller.LottoMachine;
+import lotto.domain.Lotto;
+import lotto.domain.LottoCalculator;
+import lotto.domain.LottoMatcher;
+import lotto.domain.LottoRank;
+import lotto.view.InputVIew;
+import lotto.view.ResultView;
+import java.util.List;
+import java.util.Map;
+
+public class Client {
+	public static void main(String[] args) {
+		InputVIew inputVIew = new InputVIew();
```

Comment:
코딩 컨벤션 확인 한번 부탁 드리겠습니다 😄 🙇 

```suggestion
		InputView inputView = new InputView();
```